content/kapacitor/v1.5/nodes/sideload_node.md

```diff
@@ -151,16 +153,67 @@ sideload.quiet()
 
 ### Source
 
-Source for the data, currently only `file://` based sources are supported
+Define the source of the data. The following sources are supported:
 
+- a **file URI** (`file://`)
+- a **URL** (`http://`)
+- an **endpoint name**. Use a plain string (without `file://` or `http://`) to ensure the source is interpreted as an endpoint in the `[[httpost]]` section in the Kapacitor configuration.
+
+#### Source examples
+##### File source
 
 ```js
-sideload.source(value string)
+|sideload()
+  .source('file:///path/to/dir')
+  .order('host/{{.host}}.yml', 'hostgroup/{{.hostgroup}}.yml')
+  .field('cpu_threshold', 0.0)
+  .tag('foo', 'unknown')
 ```
+##### URL source
 
-<a class="top" href="javascript:document.getElementsByClassName('article-heading')[0].scrollIntoView();" title="top"><span class="icon arrow-up"></span></a>
+```js
+|sideload()
+  .source('http://localhost:5000/threshold/')
+  .order('host/{{.host}}.yml', 'hostgroup/{{.hostgroup}}.yml')
+  .field('cpu_threshold', 0.0)
+  .tag('foo', 'unknown')
+```
+
+##### Endpoint source
+
+```js
+|sideload()
+  .source('host1')
+  .order('host/{{.host}}.yml', 'hostgroup/{{.hostgroup}}.yml')
+  .field('cpu_threshold', 0.0)
+  .tag('foo', 'unknown')
+```
 
+{{% note %}}
+A source defined as a **URL** or **endpoint** is loaded once as an HTTP GET when a task is enabled, and then on subsequent calls to the `/sideload/reload` endpoint.
+{{% /note %}}
 
+An HTTP source endpoint should return a JSON where each property is a key name specified in the order statement and its value is an object with a set of key/value pairs.
+
+```js
+
+{
+"host1": {
+"cpu_threshold": 99.9,
+"disable": "false"
+},
+"host2": {
+"cpu_threshold": 85,
+"disable": "true"
+},
+"host3": {
+"cpu_threshold": 56,
+"disable": "false"
+}
+}
+```
+
+<a class="top" href="javascript:document.getElementsByClassName('article-heading')[0].scrollIntoView();" title="top"><span class="icon arrow-up"></span></a>
```

Comment:
These don't work anymore and should probably be removed.

Reply:
Appears we have 3453 occurrences in 69 files